src/Analysis/Ast/Impl/Analyzer/PythonAnalyzer.cs

```diff
@@ -221,23 +223,21 @@ public sealed class PythonAnalyzer : IPythonAnalyzer, IDisposable {
                 var originalRemaining = walker.Remaining;
                 var remaining = originalRemaining;
                 try {
-                    _progress?.ReportRemaining(remaining);
-
                     _log?.Log(TraceEventType.Verbose, $"Analysis version {walker.Version} of {originalRemaining} entries has started.");
-                    _progress?.ReportRemaining(remaining);
                     remaining = await AnalyzeAffectedEntriesAsync(walker, stopWatch, analysisToken);
                 } finally {
                     _analysisRunningEvent.Set();
                     stopWatch.Stop();
-                    _progress?.ReportRemaining(remaining);
 
                     if (_log != null) {
-                        if (remaining == 0) {
-                            _log.Log(TraceEventType.Verbose, $"Analysis version {walker.Version} of {originalRemaining} entries has been completed in {stopWatch.Elapsed.TotalMilliseconds} ms.");
-                        } else if (remaining < originalRemaining) {
-                            _log.Log(TraceEventType.Verbose, $"Analysis version {walker.Version} has been completed in {stopWatch.Elapsed.TotalMilliseconds} ms with {originalRemaining - remaining} entries analyzed and {remaining} entries skipped.");
-                        } else {
-                            _log.Log(TraceEventType.Verbose, $"Analysis version {walker.Version} of {originalRemaining} entries has been canceled after {stopWatch.Elapsed.TotalMilliseconds}.");
+                        if (walker.Remaining == 0) {
```

Comment:
We need a log even if `walker.Remaining` is not zero

Reply:
Weird merge issue